controllers/home.js

```diff
@@ -54,18 +54,24 @@ async function home(req, res) {
 
 function getAllBreaches(req, res) {
   return res.render("top-level-page", {
-    title: "All the breaches",
+    title: "Firefox Monitor",
```

Comment:
Sometimes we wrap this in `req.fluentFormat("Firefox Monitor")` ? Whichever way we do it, let's just be consistent.

Reply:
Filed https://github.com/mozilla/blurts-server/issues/912 for this.